mmaction/models/losses/cross_entropy_loss.py

```diff
@@ -36,10 +50,33 @@ def _forward(self, cls_score, label, **kwargs):
         Returns:
             torch.Tensor: The returned CrossEntropy loss.
         """
-        if self.class_weight is not None:
-            assert 'weight' not in kwargs, "The key 'weight' already exists."
-            kwargs['weight'] = self.class_weight.to(cls_score.device)
-        loss_cls = F.cross_entropy(cls_score, label, **kwargs)
+        if cls_score.size() == label.size():
+            # cal loss for soft label
```

Comment:
One thing to take care is the `kwargs`. Please check that no kwargs are ignored silently

Reply:
cal -> calculate